shared/actions/chat/index.js

```diff
@@ -342,7 +343,7 @@ function * _loadMoreMessages (action: Constants.LoadMoreMessages): SagaGenerator
       },
       query: {
         disableResolveSupersedes: false,
-        markAsRead: true,
+        markAsRead: appFocused,
```

Comment:
Also the call to the selector should happen right above the rpc and not use a cached value